index.bs

```diff
@@ -1832,6 +1832,8 @@ To <dfn>check if cookie-based debugging is allowed</dfn> given a
 To obtain the <dfn export for=node>context origin</dfn> of a [=node=] |node|, return |node|'s [=node navigable=]'s
 [=navigable/top-level traversable=]'s [=navigable/active document=]'s [=origin=].
 
+Note: The user agent MAY return the embedded origin for specific use cases.
```

Comment:
I don't think this belongs in a note. We can instead phrase it as:

```
1. Optionally, return the embedded origin if it exists.
2. Return |node|'s ...
```

And then explain the reasoning or what an `embedded origin` is in the note.